Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -716,3 +716,593 @@ bitfinex,VSYBTC,crypto,VSYBTC,BTC,1,0.00001,0.00000001
+binance,ENJUSDT,crypto,ENJUSDT,USDT,1,0.00001000,0.10000000
+binance,MITHUSDT,crypto,MITHUSDT,USDT,1,0.00001000,0.10000000
+binance,MATICBNB,crypto,MATICBNB,BNB,1,0.00000010,1.00000000
+binance,MATICBTC,crypto,MATICBTC,BTC,1,0.00000001,1.00000000
+binance,MATICUSDT,crypto,MATICUSDT,USDT,1,0.00001000,0.10000000
+binance,ATOMBNB,crypto,ATOMBNB,BNB,1,0.00010000,0.01000000
+binance,ATOMBTC,crypto,ATOMBTC,BTC,1,0.00000010,0.01000000
+binance,ATOMUSDT,crypto,ATOMUSDT,USDT,1,0.00100000,0.00100000
+binance,ATOMUSDC,crypto,ATOMUSDC,USDC,1,0.00100000,0.00100000
+binance,ATOMPAX,crypto,ATOMPAX,PAX,1,0.00100000,0.00100000
+binance,ATOMTUSD,crypto,ATOMTUSD,TUSD,1,0.00100000,0.00100000
+binance,ETCUSDC,crypto,ETCUSDC,USDC,1,0.00100000,0.00100000
+binance,ETCPAX,crypto,ETCPAX,PAX,1,0.00100000,0.00100000
+binance,ETCTUSD,crypto,ETCTUSD,TUSD,1,0.00100000,0.00100000
+binance,BATUSDC,crypto,BATUSDC,USDC,1,0.00010000,0.01000000
+binance,BATPAX,crypto,BATPAX,PAX,1,0.00010000,0.01000000
+binance,BATTUSD,crypto,BATTUSD,TUSD,1,0.00010000,0.01000000
+binance,PHBBNB,crypto,PHBBNB,BNB,1,0.00000010,1.00000000
+binance,PHBBTC,crypto,PHBBTC,BTC,1,0.00000001,1.00000000
+binance,PHBUSDC,crypto,PHBUSDC,USDC,1,0.00001000,0.10000000
+binance,PHBTUSD,crypto,PHBTUSD,TUSD,1,0.00001000,0.10000000
+binance,PHBPAX,crypto,PHBPAX,PAX,1,0.00001000,0.10000000
+binance,TFUELBNB,crypto,TFUELBNB,BNB,1,0.00000010,1.00000000
+binance,TFUELBTC,crypto,TFUELBTC,BTC,1,0.00000001,1.00000000
+binance,TFUELUSDT,crypto,TFUELUSDT,USDT,1,0.00001000,0.10000000
+binance,TFUELUSDC,crypto,TFUELUSDC,USDC,1,0.00001000,0.10000000
+binance,TFUELTUSD,crypto,TFUELTUSD,TUSD,1,0.00001000,0.10000000
+binance,TFUELPAX,crypto,TFUELPAX,PAX,1,0.00001000,0.10000000
+binance,ONEBNB,crypto,ONEBNB,BNB,1,0.00000010,1.00000000
+binance,ONEBTC,crypto,ONEBTC,BTC,1,0.00000001,1.00000000
+binance,ONEUSDT,crypto,ONEUSDT,USDT,1,0.00001000,0.10000000
+binance,ONETUSD,crypto,ONETUSD,TUSD,1,0.00001000,0.10000000
+binance,ONEPAX,crypto,ONEPAX,PAX,1,0.00001000,0.10000000
+binance,ONEUSDC,crypto,ONEUSDC,USDC,1,0.00001000,0.10000000
+binance,FTMBNB,crypto,FTMBNB,BNB,1,0.00000100,1.00000000
+binance,FTMBTC,crypto,FTMBTC,BTC,1,0.00000001,1.00000000
+binance,FTMUSDT,crypto,FTMUSDT,USDT,1,0.00001000,0.10000000
+binance,FTMTUSD,crypto,FTMTUSD,TUSD,1,0.00001000,0.10000000
+binance,FTMPAX,crypto,FTMPAX,PAX,1,0.00001000,0.10000000
+binance,FTMUSDC,crypto,FTMUSDC,USDC,1,0.00001000,0.10000000
+binance,BTCBBTC,crypto,BTCBBTC,BTC,1,0.00000100,0.00100000
+binance,BCPTTUSD,crypto,BCPTTUSD,TUSD,1,0.00001000,0.10000000
+binance,BCPTPAX,crypto,BCPTPAX,PAX,1,0.00001000,0.10000000
+binance,BCPTUSDC,crypto,BCPTUSDC,USDC,1,0.00001000,0.10000000
+binance,ALGOBNB,crypto,ALGOBNB,BNB,1,0.00001000,0.10000000
+binance,ALGOBTC,crypto,ALGOBTC,BTC,1,0.00000010,0.01000000
+binance,ALGOUSDT,crypto,ALGOUSDT,USDT,1,0.00100000,0.00100000
+binance,ALGOTUSD,crypto,ALGOTUSD,TUSD,1,0.00100000,0.00100000
+binance,ALGOPAX,crypto,ALGOPAX,PAX,1,0.00100000,0.00100000
```

Comment:
awesome! what did you use to generate this set?

Reply:
I've written console util to fetch Binance trading symbols metadata and dump them in required format. But I would like to have an opportunity to do that within Brokerage on application start (https://github.com/QuantConnect/Lean/pull/3511#discussion_r316012763)

Reply:
Very cool! It may be useful if you post your binance console app to your github and leave a link in this PR's description. We may be able to incorporate some of your console app into a `ToolBox` app.